src/components/bookComponents/addBook.js

```diff
@@ -19,10 +19,10 @@ const Form = () => {
     dispatch(addBook(newBook));
   };
   return (
-    <form id="add-book-form" onSubmit={(e) => submitForm(e)}>
+    <form id="add-book-form" className="add-book-form" onSubmit={(e) => submitForm(e)}>
```

Comment:
- [ ] Please add the horizontal rule between the add book form and the list of books. Also, please add the missing header `ADD NEW BOOK` for this section.

Reply:
thanks